cornell-dti/carriage-web
# Pratyush/swag

### Summary

This pull request introduces swagger-ui to write API documentation for Carriage. It's a work-in-progress PR with the end goal of writing most up-to-date API documentation

### Test Plan
- run `npm run start:server` in the root.
- go to `localhost:3001/api/docs` to see the documentation.

<img width="928" alt="image" src="https://user-images.githubusercontent.com/80096811/227806371-7056aa5b-e53a-4370-8928-4cab8301fed5.png">
